### PR TITLE
Frontend tests: checks for content type

### PR DESCRIPTION
**Pull request by benilovj**
This change involves adding another config option to the test helpers to prevent an exception being raised.

This adds coverage for https://github.com/alphagov/govuk-puppet/pull/3816 .

/cc @alexmuller @mattbostock

### CODE REVIEW

File: features/frontend.feature [hunk 1]
```diff
@@ -43,7 +53,7 @@ Feature: Frontend
   Scenario: check local transactions load
     When I visit "/pay-council-tax"
     Then I should see "Pay your Council Tax"
-    When I try to post to "/pay-council-tax" with "postcode=WC2B+6SE"
```
Comment (mattbostock) [inline features/frontend.feature]: Could you put this whitespace change in a separate commit?

**Comment by mattbostock** [comment]
LGTM :+1:

**Comment by benilovj** [comment]
@mattbostock done
